src/main/scala/groundtest/Regression.scala

```diff
@@ -625,7 +625,7 @@ class MergedPutRegression(implicit p: Parameters) extends Regression()(p)
 
   io.cache.req.valid := (state === s_cache_req)
   io.cache.req.bits.cmd := M_XWR
-  io.cache.req.bits.typ := UInt(log2Ceil(64 / 8))
+  io.cache.req.bits.typ := MT_D
```

Comment:
But this is the HellaCache interface and not the TileLink interface. The L1 cache does consider the "typ" fields to be one of the MT_ constants. I'm fine with it either way, since the behavior doesn't really change. But I think using the MT_ constants is more consistent.